src/ResourceManager/AzureBackup/Commands.AzureBackup/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Deprecated Azure Backup cmdlets.
```

Comment:
Not sure what happened, please update so only your commits are shown here.

Reply:
@maddieclayton I rebased from preview, should i not have done that? and, if not, is there an easy way to undo?

Reply:
Done